### ISSUE TITLE
How do you pretrain to get the csl_stage1_weight checkpoint?

### ISSUE THREAD

**Issue by Gozier130** [ZechengLi19/Uni-Sign#25]
I reproduce the whole pre-training process for  Uni-Sign on csl-news. I use 4 H20 gpu with the exact hyperparameters provided in the paper.  After 20 epochs of training, the metrics of  csl-news test set are far lower than what you've presented above. I'm wondering what you've done to achieve the csl_stage1_weight?

**Comment by ZechengLi19** [comment]
@Gozier130 Could you please share the full log file?

**Comment by ZechengLi19** [comment]
@Gozier130 This phenomenon is normal. You can think of it as a gap between BLEU-1/2/3/4 metrics and the next-token prediction objective, which means that changes in loss do not necessarily correlate clearly with improvements or drops in these evaluation metrics. Normally, the metrics go up as the loss goes down, and then start to fluctuate within a certain range once they reach a higher performance.
